### PR TITLE
feat:add name mert

### PR DESCRIPTION
**Pull request by Kilicmert55**
OK

### CODE REVIEW

**Review by DilaZerenYilmaz — DISMISSED** [review]
Ama şampiyon TS

**Comment by Senan545** [comment]
no problem for me))

**Review by suhrab91 — DISMISSED** [review]

File: Program.cs [hunk 1]
```diff
@@ -11,7 +11,10 @@ static void Main(string[] args)
             Console.WriteLine("Yılmaz");
             Console.WriteLine("Senan");
             Console.WriteLine("Abbasov");
-            
+            Console.WriteLine("Mert");
+            Console.WriteLine("KILIÇ");
+            Console.WriteLine("Şampiyon GALATASARAY");
```
Comment (FutureBaseEditor) [inline Program.cs]: bURASI YANLIŞ OLMUŞ

**Review by Senan545 — DISMISSED** [review]
no problem for me))

**Review by FutureBaseEditor — APPROVED** [review]

**Review by DilaZerenYilmaz — APPROVED** [review]

**Review by OzanMkyn — APPROVED** [review]

**Review by Senan545 — APPROVED** [review]

**Review by suhrab91 — APPROVED** [review]

**Pull request merged**